prometheus_client/metrics.py

```diff
@@ -595,6 +595,14 @@ def __init__(self,
                  registry: Optional[CollectorRegistry] = REGISTRY,
                  _labelvalues: Optional[Sequence[str]] = None,
                  buckets: Sequence[Union[float, str]] = DEFAULT_BUCKETS,
+                 # native_hist_schema: Optional[int] = None, # create this dynamically?
```

Comment:
I believe the internal code should create the schema, and may even change the schema value on occassion.